goon/browserassets/css/browserOutput-dark.css

```diff
@@ -346,11 +346,12 @@ h1.alert, h2.alert			{color: #FFF;}
 .kidan					{color: #C64C05;}
 .slime					{color: #0077AA;}
 .drask					{color: #a3d4eb; font-family: "Arial Black";}
+.moth					{color: #869b29; font-family: "Copperplate";}
 .clown					{color: #ff0000;}
 .shadowling				{color: #Cb2799;}
 .vulpkanin				{color: #B97A57;}
 .abductor				{color: #800080; font-style: italic;}
-.mind_control				{color: #A00D6F; font-size: 3; font-weight: bold; font-style: italic;}
+.mind_control			{color: #A00D6F; font-size: 3; font-weight: bold; font-style: italic;}
```

Comment:
Phantom edit reeee

Reply:
bruh you want it being the odd one out?
![image](https://user-images.githubusercontent.com/12197162/155381641-bb420405-01c8-4c94-8dff-fe0870f6b537.png)

Reply:
![image](https://user-images.githubusercontent.com/12197162/155381698-b6c092c8-2e94-4584-9101-0b52f1e44bfa.png)

Reply:
I hate github's formatting. Make it sane for the love of god.